main/FatesInterfaceMod.F90

```diff
@@ -1954,15 +1954,15 @@ subroutine UpdateFatesRMeansTStep(sites,bc_in)
      type(ed_patch_type),  pointer :: cpatch
      type(ed_cohort_type), pointer :: ccohort
      integer :: s, ifp, io_si, pft 
-     real(r8) :: new_seedling_layer_par                ! seedling layer par in the current timestep
-     real(r8) :: new_seedling_layer_smp(maxpft)        ! seedling layer smp in the current timestep
```

Comment:
Thanks Ryan. Only question here on your change is if these means will still be able to function as pft-specific means?

Reply:
I believe the running means will remain pft-specific. These are local variables, and these variables are only used inside their pft loop, not outside.  Since the value is set inside the loop, and then used inside the loop, there is no need to remember the value for every PFT.  Does that make sense?